app/models/differential_expression_parameters.rb

```diff
@@ -0,0 +1,67 @@
+# class to hold parameters specific to differential expression jobs in PAPI
+class DifferentialExpressionParameters
+  include ActiveModel::Model
+
+  # regular expression to validate GS url format
+  GS_URL_REGEXP = %r{\Ags://}.freeze
```

Comment:
This constant might be more at home in `GoogleServiceClient`

Reply:
Maybe, but that class is very service account-specific.  This is certainly related, but really just deals with string values - it's not actually validating that the file is there.  If we end up with more parameters classes like this then I think it makes sense to abstract into a module like this.